index.html

```diff
@@ -0,0 +1,94 @@
+</head>
+<body>
+  <header>
+    <nav  class="nav-bar display-flex justify-content">
+      <div class="name">
+        <a href="#">Francis Ugorji</a>
+      </div>
+      <div class="icon-menu">
+        <img class="icon-menu" src="Images/Icon-Menu.png" alt="A Menu Bar Icon">
+      </div>
+    </nav>
+  </header>
+  <section class="headline display-flex">
+    <div class="author-info display-flex">
+      <div class="author">
+        <h1>Laura Jong</h1>
+      </div>
+      <div class="title">
+        <h2>Software Developer</h2>
+      </div>
+      <div class="description">
+        <p>I can help you build a product,<br>feature or website. Look through<br>some of my works and experience! <br>If you like what you see and have a<br>project you need coded, don't<br>hesitate to contact me.</p>
+      </div>
+    </div>
+    <div class="line"></div>
+    <section class="links display-flex justify-content">
+      <div class="languages">
+        <ul>
+          <li class="element">Language</li>
+          <li><a href="#">Javascript</a></li>
+          <li><a href="#">Ruby</a></li>
+          <li><a href="#">Html</a></li>
+          <li><a href="#">Css</a></li>
+        </ul>
+      </div>
+      <div class="frameworks">
+        <ul>
+          <li class="element">Frameworks</li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby on Rails</a></li>
+          <li><a href="#">RSPec</a></li>
+          <li><a href="#">CapyBara</a></li>
+          <li><a href="#">Selenium</a></li>
+          </ul>
+      </div>
+      <div class="skills">
+        <ul>
+          <li class="element">Skills</li>
+          <li class="skill"><a href="#">Codekit</a></li>
+          <li class="skill"><a href="#">Github</a></li>
+          <li class="skill"><a href="#">Codepen</a></li>
+          <li class="skill"><a href="#">Gitlab</a></li>
+          <li class="skill"><a href="#">Terminal</a></li>
+        </ul>
+      </div>
+    </section>
+    <div class="line"></div>
+    <section class="social-media display-flex justify-content">
+      <ul class="icons display-flex justify-content">
+        <li>
+          <a href="#"><img src="Images/Icon-GitHub.png" alt="A Github logo Icon"></a>
+        </li>
+        <li>
+          <a href="#"><img src="Images/Icon-Linkedin.png" alt="A LinkedIn logo Icon"></a>
+        </li>
+        <li>
+          <a href="#"><img src="Images/Icon-Angelist.png" alt="An Angelist logo Icon"></a>
+        </li>
+        <li>
+          <a href="#"><img src="Images/Icon-Twitter.png" alt="A Twitter logo Icon"></a>
+        </li>
+        <li>
+          <a href="#"><img src="Images/Icon-Medium.png" alt="A Medium logo Icon"></a>
+        </li>
+      </ul>
+    </section>
+    <section class="vector display-flex justify-content">
+      <img src="Images/Vector-Icon.png" alt="A Vector Icon">
+      <div>
+        <h2 class="word">Work</h2>
+      </div>
+    </section>
+  </section>
```

Comment:
For this sections, I would recommend not tu nest a `<section>` inside a `<section>` so it would be better to use `<div>` instead. Also, take care of your images, so you don't want to let them "free" inside the `<section>` may be better to use a `<div>` to wrap your images.